evalml/tests/model_understanding_tests/test_partial_dependence.py

```diff
@@ -2463,7 +2463,9 @@ def test_partial_dependence_jupyter_graph_check(
 def test_partial_dependence_fast_mode_after_dropped_feature(X_y_categorical_regression):
     """The feature selector component in this test will drop the 'time' feature, so confirm
     that fast mode handles the dropping of this feature correctly--namely that the feature
-    has no impact on predictions"""
+    has no impact on predictions
+    .
+    """
```

Comment:
This formatting doesn't look quite right

Reply:
Fixed:
https://github.com/alteryx/evalml/pull/3928/commits/2630378a3722ceac75516034279362527eff5412